src/__mocks__/reactResponsive.mock.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Note: This mock is necessary for us to be able to test for explicit screen widths in our tests using `global.innerWidth = <breakpoint size>`.